spec/models/profile_spec.rb

```diff
@@ -124,6 +124,18 @@
     end
   end
 
+  describe "of gender" do
+    it "can be 255 characters long" do
+      profile = FactoryGirl.build(:profile, :gender => "a"*255)
+      expect(profile).to be_valid
+    end
+
+    it "cannot be 256 characters" do
+      profile = FactoryGirl.build(:profile, :gender => "a"*256)
```

Comment:
- Style/HashSyntax: Use the new Ruby 1.9 hash syntax.
- Layout/SpaceAroundOperators: Surrounding space missing for operator `*`.